config.json

```diff
@@ -168,28 +161,43 @@
       "defaultValues":{
         "toolName": "msbuild",
         "settings": {
-          "MsBuildParameters":"default",
-          "MsBuildLogging":"default"
+          "MsBuildLogging":"/flp:v=normal;LogFile=clean.log"
         }
       }
     }
   },
   "tools": {
     "msbuild": {
-      "run": {
-        "windows": "Tools/msbuild.cmd",
-        "unix": "Tools/msbuild.sh"
+      "osSpecific":{
+        "windows": {
+          "defaultParameters": "/nologo /verbosity:minimal /clp:Summary /maxcpucount /nodeReuse:false /l:BinClashLogger,Tools\\net45\\Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log",
+          "path": "Tools/msbuild.cmd",
+          "filesExtension": ""
+        },
+        "unix":{
+          "defaultParameters": "/nologo /verbosity:minimal /clp:Summary /maxcpucount /l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log",
+          "path": "Tools/msbuild.sh",
+          "filesExtension": ""
+        }
       },
       "valueTypes": {
         "property": "/p:{name}={value}",
         "target": "/t:{name}",
         "internal": "/{name}"
       }
     },
-    "terminal": {
-      "run": {
-        "windows": "cmd",
-        "unix": "sh"
+    "script": {
+      "osSpecific":{
+        "windows": {
+          "defaultParameters": "",
```

Comment:
Shouldn't these properties be optional so they can be empty if they are omitted?